README.md

```diff
@@ -1,2 +1,27 @@
-# PiWeb Import Plugins
-Space to try documentation of Auto Importer plugins
+ZEISS PiWeb-Import-Sdk
+=========
+
+[![Build on develop](https://github.com/ZEISS-PiWeb/PiWeb-Import-Sdk/actions/workflows/develop.yml/badge.svg?branch=develop&event=push)](https://github.com/ZEISS-PiWeb/PiWeb-Import-Sdk/actions/workflows/develop.yml)
+[![License](https://img.shields.io/badge/License-BSD%203--Clause-blue.svg)](https://opensource.org/licenses/BSD-3-Clause)
+[![Nuget](https://img.shields.io/nuget/v/Zeiss.PiWeb.Import.Sdk?style=flat&logo=nuget)](https://www.nuget.org/packages/Zeiss.PiWeb.Import.Sdk/)
+
+The PiWeb-Import-Sdk is the basic framework for creating your own import plugins for the PiWeb Auto Importer. By creating plugins, it is possible to automate the data import of your own custom file formats or even import data from other sources like Rest-APIs or databases.
+
+### What is the PiWeb Auto Importer?
+The Auto Importer is an application of the PiWeb software solution. PiWeb applications enable the management, provision and evaluation of measurement data in the general area of industrial measurement technology. The measurement data, however, needs to be provided by other sources like measuring maschines.
+
+To make any measurement data available for evaluation in PiWeb applications, it needs to be imported into the data backend (which can be either on premises or in the cloud). The typical measuring software saves its measurement results as a file in the file system. In this case, the Auto Importer can be used to automate the import and backup of such measurement files as soon as they appear. It provides a configuration UI and can be configured to run either as an application in the background after login or fully automated as a windows service.
```

Comment:
"...either as an application in the background after login or..." (sehr detailliert) da sollten wir ggfs auf ein Unterkapitel verweisen oder auf unsere ausführliche Doku des AutoImporters.

Reply:
Oder wir lassen es einfach weg.

Reply:
Mhh ich habe es etwas gekürtzt, sollte eigentlich niemanden umbringen beim Lesen.